app/src/main/java/org/wikipedia/page/ToCHandler.java

```diff
@@ -447,6 +450,7 @@ private void scrollToListSectionByOffset(int yOffset) {
             adapter.setHighlightedSection(itemToSelect);
             currentItemSelected = itemToSelect;
         }
+        JavaScriptActionHandler.prepareToScrollTo(adapter.getItem(currentItemSelected).getAnchor(), "{ highlight: false }");
```

Comment:
Do we really need to do this here at all? The smooth thumb-scroller works just fine without it.

Reply:
This is to expand if the section scrolled to is collapsed.... so we need it here... I agree not having to scroll twice, so have made the scroll_to_anchor listener on PageFragment respond with a scroll only to references.